components/library/openssl/openssl-1.0.2/Makefile

```diff
@@ -44,8 +45,6 @@ include $(WS_MAKE_RULES)/configure.mk
 include $(WS_MAKE_RULES)/ips.mk
 include $(WS_MAKE_RULES)/lint-libraries.mk
 
-PATH=$(GCC_ROOT)/bin:/usr/bin:/usr/gnu/bin:/usr/perl5/bin
```

Comment:
Please, preserve $(GCC_ROOT)/bin in front of PATH, as OpenSSL uses compiler as just 'gcc'